Example/ScalingCarousel/CodeViewController.swift

```diff
@@ -16,6 +16,13 @@ class CodeCell: ScalingCarouselCell {
         
         mainView = UIView(frame: contentView.bounds)
         contentView.addSubview(mainView)
+        mainView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Rather than relying on the client to add this code when they override `ScalingCarouselCell`, could we move this code to a method in `ScalingCarouselCell`?

For example, we could override the `UIView` method `func didAddSubview(_ subview: UIView)`

Reply:
I have been trying 

    override open func didAddSubview(_ subview: UIView) {
        super.didAddSubview(subview)
        subview.translatesAutoresizingMaskIntoConstraints = false
        NSLayoutConstraint.activate([
            subview.leadingAnchor.constraint(equalTo: self.leadingAnchor),
            subview.trailingAnchor.constraint(equalTo: self.trailingAnchor),
            subview.topAnchor.constraint(equalTo: self.topAnchor),
            subview.bottomAnchor.constraint(equalTo: self.bottomAnchor)
            ])
    }

(And with the didAddSubview at the end, but I think it doesn't work because it gets called when the contentView is added to the Cell not when something is added to the contentView) But I think that if one user decides to create the view by code he should be responsible of applying auto layout, if not he is going to have problems with device rotation in his other views (When I add a view though code I always call an extension of the UIView that I have with this code called fillSuperView()

Reply:
Ok, sure, that's a good point about the content view being added.

We can leave it like this for the moment, and refactor in future if we find a way.

Reply:
@superpeteblaze Sounds good! :), I have planned keep using this library, if something comes to my mind related to this I will try it! :)